canvas/animate.go

```diff
@@ -0,0 +1,62 @@
+package canvas
+
+import (
+	"image/color"
+	"time"
+
+	"fyne.io/fyne"
+)
+
+const (
```

Comment:
Just a nitpick again. If we only have one `const`, we should have it on one line. Please ignore this comment if you intend to add more constants with future work.